improver/calibration/reliability_calibration.py

```diff
@@ -1270,7 +1270,6 @@ def process(self, forecast: Cube, reliability_table: Union[Cube, CubeList]) -> C
         self.threshold_coord = find_threshold_coordinate(forecast)
 
         forecast_thresholds = forecast.slices_over(self.threshold_coord)
-
```

Comment:
Would it be possible to remove these whitespace changes in reliability_calibration.py, as I don't think they're related to this PR?

Reply:
Done! I'm not sure how those changes got introduced.